packages/wallets/src/features/cfd/screens/DynamicLeverage/DynamicLeverageScreen.tsx

```diff
@@ -10,13 +11,16 @@ import './DynamicLeverageScreen.scss';
 const DynamicLeverageScreen = () => {
     const { data: dynamicLeverages } = useDynamicLeverage(PlatformDetails.mt5.platform);
     const { isDynamicLeverageVisible } = useDynamicLeverageModalState();
+    const [isDynamicLeverageHidden] = useDebounceValue(!isDynamicLeverageVisible, 600);
 
     if (!dynamicLeverages) return null;
 
     return (
         <div
             className={classNames('wallets-dynamic-leverage-screen__container', {
                 'wallets-dynamic-leverage-screen__container--flip': isDynamicLeverageVisible,
+                'wallets-dynamic-leverage-screen__container--hidden':
+                    !isDynamicLeverageVisible && isDynamicLeverageHidden,
```

Comment:
Make the component hidden and not affecting the page once its flip-away-from-view animation finishes